Mailing.rb

```diff
@@ -0,0 +1,87 @@
+require 'net/smtp'
+require 'pp'
+require 'active_record'
+require 'time'
+require 'time_difference'
+@config = YAML.load_file('config.yml')
+
+def send_time_check
+  time = Time.now.getutc
+  puts time
+end
+
+send_time_check
+
+def mail_send
```

Comment:
Split this method. Too much logic in here. 

Split into 

send_email(recepient, subject, body)
build_pr_message()
...